src/components/structures/TimelinePanel.tsx

```diff
@@ -935,25 +935,26 @@ class TimelinePanel extends React.Component<IProps, IState> {
             this.lastRMSentEventId = this.state.readMarkerEventId;
 
             const roomId = this.props.timelineSet.room.roomId;
-            const hiddenRR = SettingsStore.getValue("feature_hidden_read_receipts", roomId);
+            const sendRRs = SettingsStore.getValue("sendReadReceipts", roomId);
```

Comment:
I'm not seeing any tests updated for the flag name change: we'll need tests in order to de-labs this.